app/renderer/components/common/contextMenu.js

```diff
@@ -49,6 +49,23 @@ class ContextMenuItem extends ImmutableComponent {
   get hasAccelerator () {
     return this.accelerator !== null
   }
+
+  getNewCoordinate (e) {
```

Comment:
Although it may not be easy, this would be a great method to unit test. I think the name itself is a little vague- it might be better to rename as `getYAxis` (or similar- since it directly gets assigned to the variable with that name)